backend/src/achievements-log/entities/achievements-log.entity.ts

```diff
@@ -0,0 +1,21 @@
+import { Achievement } from 'src/achievements/entities/achievements.entity';
+import { User } from 'src/users/entities/user.entity';
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Exclude } from 'class-transformer';
+
+@Entity()
+export class AchievementsLog {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ type: 'timestamptz', nullable: true })
```

Comment:
```suggestion
  @CreateDateColumn()
```
I would consider using CreateDateColumn which automatically fills the column with the current datetime when the data is inserted into the db, saving us the trouble of doing it manually

Reply:
You're right, thanks